README.md

```diff
@@ -280,31 +281,33 @@ using the following data fields and functions:
 
 ##### Managed Resource / Ephemeral Resource / Data Source Fields
 
-|                   Field |  Type  | Description                                                                               |
-|------------------------:|:------:|-------------------------------------------------------------------------------------------|
-|                 `.Name` | string | Name of the resource/data-source (ex. `tls_certificate`)                                  |
-|                 `.Type` | string | Either `Resource` or `Data Source`                                                        |
-|          `.Description` | string | Resource / Data Source description                                                        |
-|           `.HasExample` |  bool  | Is there an example file?                                                                 |
-|          `.ExampleFile` | string | Path to the file with the terraform configuration example                                 |
-|            `.HasImport` |  bool  | Is there an import file?                                                                  |
-|           `.ImportFile` | string | Path to the file with the command for importing the resource                              |
-|         `.ProviderName` | string | Canonical provider name (ex. `terraform-provider-random`)                                 |
-|    `.ProviderShortName` | string | Short version of the rendered provider name (ex. `random`)                                |
+| Field                   | Type   | Description                                                                               |
+|-------------------------|--------|-------------------------------------------------------------------------------------------|
+| `.Name`                 | string | Name of the resource/data-source (ex. `tls_certificate`)                                  |
+| `.Type`                 | string | Either `Resource` or `Data Source`                                                        |
+| `.Description`          | string | Resource / Data Source description                                                        |
+| `.HasExample`           | bool   | Is there an example file?                                                                 |
+| `.ExampleFile`          | string | Path to the file with the terraform configuration example                                 |
+| `.HasImport`            | bool   | Is there an import shell file? (`terraform import` shell example)                         |
+| `.ImportFile`           | string | Path to the file with the command for importing the resource                              |
+| `.HasImportIDConfig`    | bool   | Is there an import terraform config file? (`import` block example with `id`)              |
+| `.ImportIDConfigFile`   | string | Path to the file with the Terraform configuration for importing the resource by `id`      |
```

Comment:
The naming/duplication is unfortunate, but these variables are an exposed interface that we should avoid breaking